Google.PowerShell/Container/GkeCmdlet.cs

```diff
@@ -18,6 +22,34 @@ public class GkeCmdlet : GCloudCmdlet
         // Service for Google Container API.
         public ContainerService Service { get; private set; }
 
+        // IAM Service used for getting roles that can be granted to a project.
+        private Lazy<ComputeService> _computeService =
+            new Lazy<ComputeService>(() => new ComputeService(GetBaseClientServiceInitializer()));
+
+        /// <summary>
+        /// Dictionary of image types with key as as tuple of project and zone
+        /// and value as the image types available in the project's zone.
+        /// This dictionary is used for caching the various image types available in a project's zone.
+        /// </summary>
+        private static ConcurrentDictionary<Tuple<string, string>, string[]> s_imageTypesDictionary =
```

Comment:
Would it be a good idea to have a timeout on these (even if that timeout is days long)?